spec.html

```diff
@@ -42188,7 +42170,7 @@ <h1>JSON.parse ( _text_ [ , _reviver_ ] )</h1>
         1. [id="step-json-parse-parse"] Let _script_ be ParseText(StringToCodePoints(_scriptString_), |Script|).
         1. NOTE: The early error rules defined in <emu-xref href="#sec-object-initializer-static-semantics-early-errors"></emu-xref> have special handling for the above invocation of ParseText.
         1. Assert: _script_ is a Parse Node.
-        1. [id="step-json-parse-eval"] Let _completion_ be the result of <emu-meta suppress-effects="user-code">evaluating _script_</emu-meta>.
+        1. [id="step-json-parse-eval"] Let _completion_ be Completion(<emu-meta suppress-effects="user-code">Evaluation of _script_</emu-meta>).
```

Comment:
We don't need the user code annotation here anymore, right? Evaluation may always invoke user code.